proc_scripts/load_data.py

```diff
@@ -3,22 +3,60 @@
+    s.setaxis('ph1',r_[0:2.]/4) #setting values of axis ph1 to line up
+    s.setaxis('ph2',r_[0:4.]/4) #setting values of axis ph1 to line up
+    s.setaxis('indirect', s.get_prop('vd'))
+    s.ft('t2',shift=True) #fourier transform
+    if fl is not None:
+        fl.next('IR prior to FTing ph')
+        fl.image(s.C.setaxis('indirect','#').set_units('indirect','scan #'))
+    s.ft(['ph1','ph2']) #fourier transforming from phase cycle dim to coherence dimension
+    s.reorder(['indirect','t2'], first=False)
+    if fl is not None:
+        s_forplot = s.C
+        fl.next('FT')
+        fl.image(s_forplot.C.setaxis('indirect','#').set_units('indirect','scan #'))
+    if fl is not None:    
+        fl.next('time domain (all $\\Delta p$)')
+        s_forplot.ift('t2')
+        fl.image(s_forplot.C.setaxis('indirect','#').set_units('indirect','scan #'))
+    if fl is not None:    
+        fl.next('frequency domain (all $\\Delta p$)')
+        s_forplot.ft('t2',pad=4096)
+        fl.image(s_forplot.C.setaxis('indirect','#').set_units('indirect','scan #'))
+    return s
 
 def proc_bruker_deut_IR_mancyc(s, fl=None):
-    logger.info("going through this")
-    s.chunk('indirect',['indirect','ph1','ph2'],[-1,4,2]) #expands the indirect dimension into indirect, ph1, and ph2. inner most dimension is the inner most in the loop in pulse sequence, is the one on the farthest right. Brackets with numbers are the number of phase cycle steps in each one. the number of steps is unknown in 'indirect' and is therefore -1.
-    s.setaxis('ph1',r_[0:4.]/4) #setting values of axis ph1 to line up
-    s.setaxis('ph2',r_[0:2.]/4) #setting values of axis ph1 to line up
+    logger.info(strm("this is the d1",s.get_prop('acq')['D'][1]))
+    if fl is not None:
+        fl.next('raw data')
+        fl.image(s)
+    s.chunk('indirect',['indirect','ph1','ph2'],[-1,2,4]) #expands the indirect dimension into indirect, ph1, and ph2. 
+                                                          #inner most dimension is the inner most in the loop in pulse sequence, 
+                                                          #is the one on the farthest right. Brackets with numbers are the number 
+                                                          #of phase cycle steps in each one. the number of steps is unknown in 
+                                                          #'indirect' and is therefore -1.
+    s.setaxis('ph1',r_[0:2.]/4) #setting values of axis ph1 to line up
+    s.setaxis('ph2',r_[0:4.]/4) #setting values of axis ph1 to line up
     s.setaxis('indirect', s.get_prop('vd'))
 #titling to coherence domain
     s.ft('t2',shift=True) #fourier transform
     s.ft(['ph1','ph2']) #fourier transforming from phase cycle dim to coherence dimension
     s.reorder(['indirect','t2'], first=False)
     if fl is not None:
         s_forplot = s.C
+        s_forplot.setaxis('indirect','#')
+        s_forplot.set_units('indirect','scan #')
```

Comment:
I would string the methods into a single command line -- I say this in case you don't know that you can do that.  Otherwise, this could be a matter of personal style

`s.C.setaxis(....).set_units(...)`